apps/protailwind/src/pages/tutorials/[module]/[exercise]/solution.tsx

```diff
@@ -23,22 +32,33 @@ export const getStaticPaths: GetStaticPaths = async (context) => {
   const paths = tutorials.reduce((acc: any[], tutorial: any) => {
     return [
       ...acc,
-      ...tutorial.exercises.map((exercise: any) => {
-        return {
-          params: {
-            module: tutorial.slug.current,
-            exercise: exercise.slug,
-          },
-        }
-      }),
+      ...tutorial.exercises
+        .filter((exercise: Exercise) => Boolean(exercise.solution))
```

Comment:
filter out any exercise that doesn't have a solution! since we have a "bonus" exercise that is similar to an "outro" we don't want to build it if it doesn't have a solution.

this wasn't a problem previously because we weren't trying to serialize the transcript and videoResourceId, which are undefined and next won't deal with them